core/contracts/common/FixedPoint.sol

```diff
@@ -56,6 +71,21 @@ library FixedPoint {
         return fromUnscaledUint(a).rawValue < b.rawValue;
     }
 
+    /** @dev Whether `a` is less than or equal to `b`. */
+    function isLessThanOrEqual(Unsigned memory a, Unsigned memory b) internal pure returns (bool) {
```

Comment:
Thanks for adding these! While we're here, would you mind adding an `isEqual` that takes a `uint` as the second argument?

Reply:
done